HUD/RemoveBlip.md

```diff
@@ -5,12 +5,12 @@ ns: HUD
 
 ```c
 // 0x86A652570E5F25DD 0xD8C3C1CD
-void REMOVE_BLIP(Blip* blip);
+void REMOVE_BLIP(int blip);
```

Comment:
```suggestion
void REMOVE_BLIP(Blip* blip);
```

Reply:
No it's required, like you can see here docs doesn't handle correctly the params:
![image](https://github.com/user-attachments/assets/a57957f8-fee5-4844-8b2c-0c238f86a73f)

This will fix the visual:
![image](https://github.com/user-attachments/assets/e8037ef5-aa09-4749-9a31-7394aa5c4f15)


Old issue, **this was requested by the team before making the PR**.

Reply:
This doesn't "fix" it, this works around it, and subsequently breaks ABI compatibility with C# **which you cannot do**.